pkg/apic/provisioning/credentialrequest.go

```diff
@@ -10,4 +10,6 @@ type CredentialRequest interface {
 	GetCredentialDetailsValue(key string) string
 	// GetCredentialType returns the type of credential related to this request
 	GetCredentialType() string
+	// GetCredentialData returns the map[string]interface{} of data from the request
+	GetCredentialData() map[string]interface{}
```

Comment:
@tsjohns9 this change ant he GetAccessRequestData above are meant to return the data that matches the request schema in the *RequestDefinition resources